packages/debugger/test/stacktrace.js

```diff
@@ -126,8 +134,9 @@ describe("Stack tracing", function () {
     try {
       await instance.run(0); //this will throw because of the revert
     } catch (error) {
-      txHash = error.hashes[0]; //it's the only hash involved
+      txHash = error.receipt.transactionHash;
     }
+    if (!txHash) assert.fail("should have thrown and set txHash");
```

Comment:
And another one of these; there's more in this file of course but I won't mark them each individually.